src/lib.rs

```diff
@@ -0,0 +1,191 @@
+use std::collections::HashMap;
+// use std::path::PathBuf;
+use std::{env, fs};
+use zed::settings::LspSettings;
+use zed_extension_api::{self as zed, serde_json, Result};
+
+struct EmberExtension {
+    did_find_server: bool,
+}
+
+const SERVER_PATH: &str =
+    "node_modules/@ember-tooling/ember-language-server/bin/ember-language-server.js";
+
+impl EmberExtension {
+    fn server_exists(&self) -> bool {
+        fs::metadata(SERVER_PATH).map_or(false, |stat| stat.is_file())
+    }
+
+    fn server_script_path(&mut self, id: &zed::LanguageServerId) -> Result<String> {
+        let node_dependencies: HashMap<&str, &str> = [
+            ("@ember-tooling/ember-language-server", "2.30.3"),
+            ("els-addon-file-watcher", "0.0.2"),
+            ("els-addon-glint", "0.6.4"),
+            ("els-a11y-addon", "1.0.4"),
+            ("ember-fast-cli", "1.3.1"),
+            ("els-intl-addon", "1.0.3"),
```

Comment:
ELS has builtin els-intl-addon, so, we could skip this one

Reply:
Thanks, I was meaning to look into which addons to include by default & whether to version them or always pull the latest ones!